source/vst/vstcomponentbase.cpp

```diff
@@ -81,8 +81,13 @@ tresult PLUGIN_API ComponentBase::terminate ()
 	if (peerConnection)
 	{
 		peerConnection->disconnect (this);
-		peerConnection->release ();
-		peerConnection = nullptr;
+		
+		// peerConection might become null during the disconnect() above
```

Comment:
Why do you think that this can happen ? I don't see where it could be set to nullptr anywhere else.